helm_deploy/values-dev.yaml

```diff
@@ -0,0 +1,25 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
```

Comment:
Does this ticket include the HPA stuff?

Reply:
No. As we're using the service template, this PR goes slightly beyond the scope of MAS-64

Reply:
> Keep the autoscaler, 1/2 for dev/preprod 2/4 for prod